etcd-manager/cmd/etcd-manager/main.go

```diff
@@ -202,6 +220,14 @@ func RunEtcdManager(o *EtcdManagerOptions) error {
 		return fmt.Errorf("backup-store is required")
 	}
 
+	if o.NetworkCIDR == "" {
+		o.NetworkCIDR = os.Getenv("ETCD_MANAGER_NETWORK_CIDR")
+	}
```

Comment:
Let's remove the env var for now completely.

Reply:
Whoops, I thought I removed it.